db/migrate/20150811020817_create_transactions.rb

```diff
@@ -0,0 +1,11 @@
+class CreateTransactions < ActiveRecord::Migration
+  def change
+    create_table :transactions do |t|
+      t.integer :amount, default: 0, null: false, index: true
```

Comment:
I'm not sure if setting a default of 0 makes sense, since in the app we only allow values greater than 0. There are no other values that make sense either, so I think I'd rather not have default.